interpreter/text/parser.mly

```diff
@@ -566,42 +566,58 @@ func_body :
 
 /* Tables, Memories & Globals */
 
+table_ref :
+  | LPAR TABLE var RPAR { fun c -> $3 c }
+
 offset :
   | LPAR OFFSET const_expr RPAR { $3 }
   | expr { let at = at () in fun c -> $1 c @@ at }  /* Sugar */
 
+elem_kind :
+  | FUNC { FuncRefType }
+
 elemref :
```

Comment:
While you're here, can you rename this to `elem_expr` (and `elem_expr_list` below)?

Reply:
Done.